lib/net/nforceif/src/sys_arch.c

```diff
@@ -118,7 +117,7 @@ sys_thread_new(const char *name, lwip_thread_fn function, void *arg, int stacksi
     abort();
   }
 
-  st->handle = XCreateThread((void *)function, arg, NULL);
+  st->handle = CreateThread(NULL, stacksize, (void *)function, arg, 0, NULL);
```

Comment:
`sys_thread.handle` is still an `int` but it should be `HANDLE` now.

Reply:
Oops, missed that. Fixed.